.github/workflows/go.yml

```diff
@@ -0,0 +1,32 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
```

Comment:
nit: I wonder if ubuntu-slim would work? Might be a little lighter than ubuntu-latest

Reply:
Turns out that only the following are supported:

Windows Server 2019 | windows-latest or windows-2019
Ubuntu 18.04 | ubuntu-latest or ubuntu-18.04
Ubuntu 16.04 | ubuntu-16.04
macOS Catalina 10.15 | macos-latest or macos-10.15


We could run our own runner at some point, but these are ones github supports.